Core/LocalAdmin.cs

```diff
@@ -126,7 +125,7 @@ internal async Task Start(string[] args)
             if (OperatingSystem.IsWindows())
             {
                 ConsoleUtil.WriteLine("Such error should never occur on Windows.", ConsoleColor.Red);
-                ConsoleUtil.WriteLine("Open issue on the LocalAdmin GitHub repository (https://github.com/northwood-studios/LocalAdmin-V2/issues) or contact our technical support!", ConsoleColor.Red);
+                ConsoleUtil.WriteLine("Please open an issue on the LocalAdmin GitHub repository: https://github.com/northwood-studios/LocalAdmin-V2/issues or contact our technical support!", ConsoleColor.Red);
```

Comment:
The previous form was in my opinion better. The colon doesn't really make sens if we have "or contact" later in the same sentence.